thirdparty/build-thirdparty.sh

```diff
@@ -566,6 +566,19 @@ build_flatbuffers() {
   cp libflatbuffers.a $TP_LIB_DIR/libflatbuffers.a
 }
 
+build_brotli() {
+    check_if_source_exist $BROTLI_SOURCE
+    cd $TP_SOURCE_DIR/$BROTLI_SOURCE
+    mkdir -p $BUILD_DIR
+    cd $BUILD_DIR
+    ${CMAKE_CMD} .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR
+    ${CMAKE_CMD} --build . --config Release --target install
```

Comment:
```suggestion
    ${CMAKE_CMD} .. -G "${CMAKE_GENERATOR}" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR
    ${BUILD_SYSTEM} -j$PARALLEL
    ${BUILD_SYSTEM} install
```

Reply:
if we can keep consistent with other dependency build command. support both make and ninja and takes `$PARALLEL` in consideration.